FP_Factions/gordon_reskins/boonie/config.cpp

```diff
@@ -130,8 +130,8 @@ class CfgPatches
 			"FP_GOR_H_Booniehat_sumrak_hs",
 			"FP_GOR_H_Booniehat_syrian",
 			"FP_GOR_H_Booniehat_syrian_hs",
-			"FP_GOR_H_Booniehat_tochka",
-			"FP_GOR_H_Booniehat_tochka_hs"

```

Comment:
Why remove those? they are still present in config. MakePbo will add them back regardless.

Reply:
that was a fucky wucky on my part, fixed